src/cli/commands/combine.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import {Command} from "@oclif/command";
+import fs from "fs";
+import stringify from "json-stable-stringify";
+import {promisify} from "util";
+
+import {Graph} from "../../core/graph";
+
+export default class CombineCommand extends Command {
+  static description = "combine multiple contribution graphs into one big graph";
+
+  // We disable strict-mode so that we can accept a variable number of
+  // arguments. Although this is what the docs [1] suggest, it doesn't
+  // seem like the best way to do it. TODO: Investigate.
+  //
+  // [1]: https://oclif.io/docs/commands.html
```

Comment:
We could file an issue on oclif asking for a way to do this without disabling strict.

Reply:
Agreed; I was thinking the same thing. This seems okay to me for now, though.